src/universal/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.js

```diff
@@ -36,6 +36,8 @@ class OutcomeCardContainer extends Component {
     super(props);
     const {outcome: {content}} = props;
     this.state = {
+      isEditingText: false,
+      isEditingMeta: false,
```

Comment:
As in, whenever the number of editing clients moves from 0 to 1 or 1 to 0, we can fire that query :)